migrations/20190626000000_form_db.js

```diff
@@ -0,0 +1,25 @@
+exports.up = knex =>
+  knex.schema.alterTable('form', table => {
+    table
+      .integer('sequence_no')
+      .notNullable()
+      .defaultTo(1)
+    table.bigInteger('booking_id').notNullable()
+    table
+      .timestamp('start_date')
+      .notNullable()
+      .defaultTo(knex.fn.now(6))
+
+    table.string('user_id', 32).alter()
+    table.string('status', 20)
```

Comment:
this could potentially be an enum but don't really know the final set of states and migrating enums is a bit fiddly in knex